setup.py

```diff
@@ -59,6 +59,16 @@
         # TODO(niboshi): Consider upgrading pytest to >=3.6
         'pytest-timeout<1.3.0',
     ],
+    'jenkins': [
+        '-r test',
+        # pytest-timeout>=1.3.0 requires pytest>=3.6.
+        # TODO(niboshi): Consider upgrading pytest to >=3.6
+        'pytest-timeout<1.3.0',
+        'pytest-cov',
+        'nose',
```

Comment:
* `nose` is no longer necessary to run Chainer tests.
* How about adding `attrs` restrictions as well? https://github.com/chainer/chainer-test/pull/529

Reply:
How about separate PRs?
The purpose of this PR is not to change requirements.

Reply:
Ok.